libblkid-rs-sys/build.rs

```diff
@@ -1,8 +1,31 @@
 use bindgen::Builder;
 
-use std::{env, path::PathBuf};
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+#[cfg(target_os = "macos")]
+fn fix_include(include: &PathBuf) -> &Path {
+    if include.ends_with("include/blkid") {
+        include.parent().unwrap()
+    } else {
+        include
+    }
+}
+
+#[cfg(not(target_os = "macos"))]
+fn fix_include(include: &PathBuf) -> &Path {
+    include
+}
 
 fn main() {
+    #[cfg(target_os = "macos")]
+    env::set_var(
+        "PKG_CONFIG_PATH",
+        "/opt/homebrew/opt/util-linux/lib/pkgconfig",
+    );
```

Comment:
After some Googling, I have determined that this is a problem beyond the scope of this library. This seems to be a regular condition of using homebrew and pkg-config together. I am happy to add a section to the documentation for people using it on MacOS but I do not think we should set environment variables for users in this way. In the case where two directories need to be specified, the user supplied one will be ignored and overwritten by this. I do not think we should set the variables for users.

Reply:
Makes sense, I'll remove it.